tools/src/wyvern/stdlib/support/RandomWrapper.java

```diff
@@ -0,0 +1,19 @@
+package wyvern.stdlib.support;
+import java.util.Random;
+
+public class RandomWrapper {
```

Comment:
Is this needed?  If not, please omit it from the PR.  It doesn't seem to be used.

Reply:
Hi Professor Aldrich

My linkedlist tests uses this. 

Do you want me to omit it and pull again?

Thanks!

Reply:
I see.  But your linkedlist tests are not part of the PR right now, correct?  So you should either add the tests to the PR, or remove the RandomWrapper from the PR.